### PR TITLE
#283 iframe page for embedding in 3rd party websites - continued

### PR DESCRIPTION
**Pull request by webaissance**
* Remark out region_submission.populate_boundaries which was causing an error - hope this will fix the issue

### CODE REVIEW

**Comment by webaissance** [comment]
@mattsayre - this Pull Request is waiting for approval so we can merge it. It's a tiny change that should fix the issue.

**Review by mattsayre — APPROVED** [review]

**Comment by webaissance** [comment]
@mattsayre and @Kourchenko  - this went to Test and it worked! The bug is resolved and the test Regional site is now working as designed. We can confirm on https://test.fasterinternetoregon.org/  It works!

**Pull request merged**
